src/components/MediaObject/MediaObject.js

```diff
@@ -69,6 +71,10 @@ MediaObject.defaultProps = {
 };
 
 MediaObject.propTypes = {
+  /**
+   * Renders a customTag instead of a div
```

Comment:
maybe `customTag` better to right as `custom tag` because it is not a prop name anymore